eth/tracers/native/prestate.go

```diff
@@ -82,6 +74,7 @@ func (t *prestateTracer) CaptureStart(env *vm.EVM, from common.Address, to commo
 	// The sender balance is after reducing: value, gasLimit, intrinsicGas.
 	// We need to re-add them to get the pre-tx balance.
 	fromBal := hexutil.MustDecodeBig(t.prestate[from].Balance)
+	intrinsicGas := t.gasLimit - gas
```

Comment:
The consumedGas can be always equivalent to `t.gasLimit` right?

Reply:
Right, I was doing `gasLimit - gas + gas` haha